app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -365,9 +365,13 @@ def api_version
       @api_version ||= client.api_version
     end
 
-    def ca_cert_store(cert)
-      return if cert.blank?
-      OpenSSL::X509::Store.new.add_cert(OpenSSL::X509::Certificate.new(cert))
+    def ca_cert_store(certs)
+      return if certs.blank?
+      store = OpenSSL::X509::Store.new
+      certs.split("-----END CERTIFICATE-----").each do |cert|
```

Comment:
You can make the split pattern as part of the result like this:

```ruby
certs.split(/(?=-----BEGIN)/)
```

Also, I'm not sure how big chance is the certs would start with `-----BEGIN TRUSTED CERTIFICATE-----` (I saw that format in my pki directory), but keeping the pattern shorter `-----BEGIN` can potentially cover more cases than the specific version that is there right now.